pkg/cmd/pulumi/import_test.go

```diff
@@ -164,3 +227,66 @@ func TestParseImportFile_errors(t *testing.T) {
 		})
 	}
 }
+
+func TestParseImportFileSameName(t *testing.T) {
+	t.Parallel()
+	f := importFile{
+		Resources: []importSpec{
+			{
+				Name:    "thing",
+				ID:      "thing",
+				Type:    "foo:bar:bar",
+				Version: "0.0.0",
+			},
+			{
+				Name:    "thing",
+				ID:      "thing",
+				Type:    "foo:bar:bar",
+				Version: "0.0.0",
+			},
+		},
+	}
+	imports, _, err := parseImportFile(f, false)
```

Comment:
I feel like this test ought to check what names _have_ been given, not just that we didn't make any duplicates. We could have all unique names because we appended GUIDs to the end of everything but that's not what we want.